src/PagesBody/ClassChoicePage/ClassChoicePage.js

```diff
@@ -0,0 +1,47 @@
+import { useContext, useState } from 'react';
+
+import CLASSES from '../../Consts/classesConst';
+import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
+import { ContestContext } from '../../Context/ContestContext';
+import FakeButton from '../../Atoms/FakeButton/FakeButton';
+import MainButton from '../../Atoms/MainButton/MainButton';
+
+const ClassChoicePage = () => {
+  const { contestState } = useContext(ContestContext);
+  const { contestId } = contestState;
+  console.log(contestState);
+  const classesArr = CLASSES.map((elem) => elem.name);
```

Comment:
You are reading here CLASSES from classesConst.
You must read classes from contests
Contest can have for example only 2 classes.
You can use here this code from my PR:

You can change your classesArr to this:
![image](https://user-images.githubusercontent.com/75560322/153422668-d2d12530-186c-4fe7-af98-022a6a0093cc.png)


![image](https://user-images.githubusercontent.com/75560322/153422546-e0b5ea80-9dcf-41ea-a737-a2f7e864f7bd.png)

Reply:
updated